### ISSUE TITLE
8px off the top

### ISSUE THREAD

**Issue by stephenfeather** [jesusoterogomez/react-notify-toast#44]
Great little add on.  Well implemented as well.  Not needing to pass context or props around to reach the primary <Notification/> placement was excellent thinking.

Testing in Chrome Dev using React 16, the toast is sitting 8px down from the top.  If I hack the bin and set the y transform to -100 instead of -108, its fixed.

What was the original thinking behind the 108 offset? Want to be sure I'm not going to run into any side effects you already dealt with before setting to 108.

Thanks.

**Comment by jesusoterogomez** [comment]
Thanks for the kind words!
I've seen your comment in the other post as well, and I have to admit that I really hate this `magic` hardcoded value, and that I really don't remember the reasoning behind it, as I started this project as a really small add-on to a really old project in Angular and when I ported it, this value was already there.

I have been struggling with multiline notifications this week, so I am thinking of getting rid of this altogether.

I don't have a lot of time to implement it soon, but my idea is:
1. Load and measure height of the content into the notification's DOM element
2. Use that height for the `translateY` animations.
3. ???
4. Profit!

This would make every `translateY` dynamic to the content of each notification, and I believe it would solve the multiline problem.
